src/utils/getTestFrameworkName.ts

```diff
@@ -2,12 +2,12 @@
 import type { PackageJSON } from "@nodesecure/npm-types";
 
 // CONSTANTS
-const kNodeTestRunnerRegex = /(?:node\s+--test|tsx)/;
+const kNodeTestRunnerRegex = /[node|tsx]\s+.*--test/;
 
 export function getTestFrameworkName(packageJson: PackageJSON): string {
   const { devDependencies: deps = {}, scripts = {} } = packageJson;
 
-  if (kNodeTestRunnerRegex.test(scripts.test)) {
+  if (kNodeTestRunnerRegex.test(scripts.test) || kNodeTestRunnerRegex.test(scripts["test-only"])) {
```

Comment:
As discussed this can be improved, depends if you're down to go futher (cf [my first comment](https://github.com/dashlog/core/pull/96#issuecomment-2585308130)).
Otherwise I guess it's fine for our own projects and/or it can be improved in the future

Reply:
I would be tempted to try to do it as we discussed.